modules/fts/pages/fts-query-string-syntax-field-scoping.adoc

```diff
@@ -14,9 +15,11 @@ You can adjust this by prefixing items with `+` or `-`.
 * Prefixing with `+` places that item in the MUST portion of the boolean query.
 * Prefixing with `-` places that item in the MUST NOT portion of the boolean query.
 
+Note, the above syntax is a simple and easy way to create xref:fts-supported-queries-compound-query.adoc[Compound Queries] in Search.
+
 === Example
 
 For example, `description:pool` performs a xref:fts-supported-queries-match.adoc[match query] for the term `pool`, in the `description` field.
 
 For example, `+description:pool -continental breakfast` performs a boolean query that MUST satisfy the match query for the term `pool` in the `description` field, MUST NOT satisfy the match query for the term `continental` in the `default` field, and SHOULD satisfy the match query for the term `breakfast` in the `default` field.
-Result documents satisfying the SHOULD clause score higher than those that do not.
\ No newline at end of file
+Result documents satisfying the SHOULD clause score higher than those that do not.
```

Comment:
So I want to make sure I understand this correctly: a document that matches: 

- Pool in the description field 
- No continental in the default field 

will be scored as less than another document that also has breakfast in the default field?

Reply:
I asked a few questions of Abhi about this I will get back to you.

Reply:
**First**

On this page https://docs.couchbase.com/server/current/fts/fts-query-string-syntax-field-scoping.html the"default" field I believe means it is included in _all.

For basic indexes that we just let index everything we add everything to the _all catch all - right?
Of course when we fine tune our index we can prevent fields or subdocs from being in the the _all catch all.

This nomenclature of the default field on this page is a bit confusing.

**Second**

Now Sarah your question about this page:

> So I want to make sure I understand this correctly: a document that matches:
> Pool in the description field
> No continental in the default field
> will be scored as less than another document that also has breakfast in the default field? 

I believe scores don't stay the same and you can not add and subtract from a prior query (i.e. that is not identical) since the range of scores are unique to a specific query.

`+description:pool -continental breakfast
`

Gives 46 results and the top result is hotel_20420 with a score of [3.7529735330139866]  and hotel_5848 a min score of [0.778778750224265]

and

`+description:pool -continental
`

Also gives 46 results and the top result is hotel_20420 with a score of [4.235059965679149] and hotel_33884 a min score of [0.8150376911073436]

Thus since the queries are different we have different ordering and different score ranges.

A bit more detail to add similar to your question in 7.2's PR

The query string query is a boolean query - which has must, must_not and should components.

- Anything prefixed with + goes into the must conjunction
- Any prefixed within -  goes into the must_not disjunction
- Any NOT prefixed with + /- will go into the should disjunction

When must and/or must_not is defined, a should is meant to affect scoring of documents only.

When both must and must_not are not defined, a should simply is seen as a disjunction query.

Reply:
I think we can leave clearing this up to a separate PR. Thanks for the info :)